### DISCUSSION TITLE
เสนอ Feedback สำหรับ Notion ของ Thai Developer Portal กัน

### DISCUSSION THREAD

**Discussion by heypoom** [kaogeek/hack-th#2] [Announcements]
มาเสนอ feedback เกี่ยวกับโครงสร้างข้อมูลในเว็บ Hack.go.th (Thailand Developer Portal) กัน!

ตอนนี้พวกเราใช้ [Notion](https://jirachai.notion.site/Hack-go-th-Wiki-d269dc5943934c23a7ef7427537fcea2?pvs=4) เพื่อทำเป็นตัวอย่างข้อมูลไว้ฮะ ว่าใน Thai Developer Portal จะมีข้อมูลอะไรบ้าง

แนะนำกันได้นะครับ ว่า structure น่าจะเป็นแบบไหน :)

- [Link to Notion prototype](https://jirachai.notion.site/Hack-go-th-Wiki-d269dc5943934c23a7ef7427537fcea2?pvs=4)
- [คำอธิบาย prototype](https://www.notion.so/jirachai/About-This-MVP-634a27a132554fe594b89167b4aa1d17?pvs=4)

**Comment by zivaja** [comment]
ตอนนี้เลือก Stack กันแล้วใช่มั้ยครับ มี workflow ไปที่ไหนแล้วบ้างรึเปล่าครับ
กำลังคิดว่า components พวกนี้จะอยู่ที่ไหนบ้าง
- Post Content
- Static Files (Image, Video, Document)
- ETC. OpenAPI, Swgger, Postman Collection
